src/containers/notFound/NotFound.jsx

```diff
@@ -0,0 +1,16 @@
+import React, { Component } from 'react'
+import { connect } from 'react-redux'
+import ErrorDisplayer from '../../commons/errorDisplayer/ErrorDisplayer'
+
+class NotFound extends Component {
```

Comment:
Why do you need a container ? You can just have a component, as other common (utils) components.

Reply:
deleted